portal/src/main/webapp/WEB-INF/jsp/patient_view/patient_view.jsp

```diff
@@ -1155,5 +1155,36 @@ window["<%=PatientView.CANCER_STUDY_META_DATA_KEY_STRING%>"]
         = <%=jsonMapper.writeValueAsString(request.getAttribute(PatientView.CANCER_STUDY_META_DATA_KEY_STRING))%>;
 
 </script>
+<!-- Add script and style files for cbioportal-frontend -->
+<script>
+// Set frontend route to /patient
+window.defaultRoute = "/patient";
+// get root URL
+<%
+String url = request.getRequestURL().toString();
+String baseURL = url.substring(0, url.length() - request.getRequestURI().length()) + request.getContextPath();
+baseURL = baseURL.replaceFirst("http.*://", "");
+%>
+__API_ROOT__ = "<%= baseURL %>/api-legacy";
+
+if (localStorage.getItem('localdev') === "true") {
+    // Use cbioportal-frontend localhost:3000 for dev
+    document.write('<script src="http://localhost:3000/reactapp/js/common.bundle.js"></scr'+'ipt>');
+    document.write('<script src="http://localhost:3000/reactapp/js/main.app.js"></scr'+'ipt>');
+    // Show alert
+    document.write('<div style="position: fixed; top: 0; left: 0; width: 100%;">'+
+                       '<div class="alert alert-warning">' +
+                            '<button type="button" class="close" data-dismiss="alert">&times;</button>' +
+                            'cbioportal-frontend dev mode' +
+                       '</div>' +
+                   '</div>');
+} else {
```

Comment:
@alisman Seems like https://github.com/cBioPortal/cbioportal/pull/1846 has been merged, so I think we should just go ahead and show the replaced clinical information tab. Even though there is slight regression in terms of user features I think it's a good idea to get something replaced in the next release. Just so we can get used to the process of adding new features from the frontend repo to the parent project.